frame/society/src/lib.rs

```diff
@@ -681,9 +699,40 @@ impl<T: Trait> Module<T> {
 		// TODO: Clean up vouching
 	}
 
-	/// `kind` lets you know whether there's either a deposit at stake or a member's vouching rights.
-	fn suspend_candidate(_who: &T::AccountId, _kind: BidKind<T::AccountId, BalanceOf<T>>) {
-//		unimplemented!();
+	/// Pay an accepted candidate their bid value.
+	fn pay_accepted_candidate(
+		candidate: &T::AccountId,
+		value: BalanceOf<T>,
+		kind: BidKind<T::AccountId, BalanceOf<T>>
+	) {
+		let maturity = Self::maturity();
```

Comment:
wow.
amazing. you've turned decoding the members from an `1x` in my original code to `(N+2)x` in your code. this is obviously brain-dead.